### PR TITLE
New BrainHack project  "General Spinal Cord Segmentation"

### PR DESCRIPTION
**Pull request by louisfb01**
This is a pull request to add [my BrainHack project](https://github.com/brainhack-school2023/bouchard_project) to the gallery, following https://school.brainhackmtl.org/project_guide/.

### CODE REVIEW

**Comment by louisfb01** [comment]
@valosekj

**Comment by valosekj** [comment]
> Tagging TA @valosekj for review :)

Nice! I have a vacation from 9-17 June --> will review it after.

File: content/en/project/general-spinal-cord-segmentation/index.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+
+### Objectives
+
+* Create a script for converting N BIDS datasets into one large “nnU-Net-ready” dataset train/test split. Read more [here](https://github.com/brainhack-school2023/bouchard_project/tree/main/docs).
+* Compare a general model to specific models for spinal cord segmentation on the datasets’ test sets. Results shown [here](https://github.com/brainhack-school2023/bouchard_project/blob/main/results/results_analysis.ipynb).
+* Test the resulting 2 frameworks on a fifth and sixth dataset to demonstrate the generalizability capabilities of the larger model. Results shown [here](https://github.com/brainhack-school2023/bouchard_project/blob/main/results/results_analysis.ipynb).
+
+### Deliverables
+
+* A Github repository with codes and scripts to reproduce training and testing `github.com/brainhack-school2023/bouchard_project`.
```
Comment (pbellec) [inline content/en/project/general-spinal-cord-segmentation/index.md:87]:
```suggestion
* A [Github repository](https://github.com/brainhack-school2023/bouchard_project) with codes and scripts to reproduce training and testing.
```

Comment (pbellec) [reply content/en/project/general-spinal-cord-segmentation/index.md:87]: There are several references to docs throughout the report which would be better formatted as links. If you want to have the links in plain text as well, you can make links like [github.com/bla](https://github.com/bla]. These will be rendered in the same style as what you used, but they will be clickable.

**Review by pbellec — CHANGES_REQUESTED** [review]
I've rendered the page, and everything looks great. Congrats on the awesome project. Pending the minor comment I'll left, I'll be ready to merge. I'll wait a bit on the review by the Poly hub though.

**Review by pbellec — APPROVED** [review]

**Comment by pbellec** [comment]
The review was never properly completed but the report looks good, so I am merging.

**Comment by valosekj** [comment]
Apologies for no review from my side; it somehow slipped through the cracks.

Indeed, the project looks good. Thanks for merging!

**Pull request merged**